### PR TITLE
For #8: Speel checked with grammarly.

### PR DESCRIPTION
**Pull request by denispasin**
# Context

#8 

## Related tickets

#8 Speel check Readme.

# What's inside

- [x] Spellcheck

# Checklist:

- [x] I have added tests: No but as expected
- [x] I have made corresponding changes to the documentation: Indeed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ curl www.bar-app.docker/api/v1/baz_service/qzz
 
 ### dip dns
 
-Runs DNS server container based on https://github.com/aacebedo/dnsdock It used for container to container requests through nginx. An application's docker-compose.yml should define `dns` configuration with environment variable `$DIP_DNS` and connect to external network `frontend`. `$DIP_DNS` will be automatically assigned by dip.
+It runs a DNS server container based on https://github.com/aacebedo/dnsdock. It is used for container to container requests through Nginx. An application's docker-compose.yml should define `DNS` configuration with environment variable `$DIP_DNS` and connect to external network `frontend`. `$DIP_DNS` will be automatically assigned by dip.
```
Comment (bibendi) [inline README.md]: `dns` should be lowercased here

Comment (bibendi) [inline README.md]: I think there is no need `It` at the beginning

Comment (denispasin) [reply README.md]: If I remove the `It` Grammarly complain.
Capture d’écran, le 2019-08-13 à 16 39 59

Comment (bibendi) [reply README.md]: Robots can also be wrong :) The above sentences without `It` too and it doesn't complain.

Comment (denispasin) [reply README.md]: I'll remove it :) (I don't really have a strong opinion)

**Pull request merged**